recipes/wdiff/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "wdiff" %}
+{% set version = "1.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 34ff698c870c87e6e47a838eeaaae729fa73349139fc8db12211d2a22b78af6b
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+    - texinfo
+  host:
+    - ncurses  # [linux]
```

Comment:
Isn't this needed on macOS as well?

Reply:
build works on macOS without, so I didn't add it.
Should I?

Reply:
Yeah because it is probably linking to the system library, which will cause issues if macOS updates that in different versions

Reply:
Indicated by the following log output:
```
   INFO (wdiff,bin/wdiff): Needed DSO /Applications/Xcode_12.4.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk/usr/lib/libncurses.5.4.dylib found in $SYSROOT
```

Reply:
Removed the selector, thanks all!